app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -43,6 +51,9 @@ class DeadlineDetailsActivity : AppCompatActivity() {
     @Inject
     lateinit var clockService: ClockService
 
+    @Inject
+    lateinit var firebasePdfRepository: PdfRepository
```

Comment:
Nitpick but the pdf repository isn't necessarily linked to firebase, since we mock it in tests.
The fact that it is a firebase implementation should be irrelevant to the activity.
It would also be more consistent with the rest of the code.